draft-thomson-webpush-protocol.xml

```diff
@@ -725,11 +864,13 @@ DATA         [stream 4] +END_STREAM
          protocols necessary to build and support those relationships.
         </t>
         <t>
-         A "subscription", "push" or push message URI therefore acts as a bearer
-         token.  Knowledge of a "subscription" URI implies authorization to
-         either receive push messages or delete the subscription. Knowledge of a
-         "push" URI implies authorization to send push messages.  Knowledge of a
-         push message URI allows for reading that specific message<!-- and
+         A "subscription", "push", "receipt", or push message URI therefore acts
+         as a bearer token.  Knowledge of a "subscription" URI implies
+         authorization to either receive push messages or delete the
+         subscription. Knowledge of a "push" URI implies authorization to send
+         push messages.  Knowldege of a "receipt" URI implies authorization to 
+         receive push receipts. Knowledge of a push message URI allows for
+         reading that specific message<!-- and
          enables message acknowledgment -->.
```

Comment:
You need to use what I put in the comment.  Note that this means that the application server can acknowledge its own messages, since it is the other entity that learns the push message resource URI.